ci-operator/jobs/coreos/coreos-assembler/coreos-coreos-assembler-main-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
   coreos/coreos-assembler:
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
Just to clarify, we do need to set `always_run` to false in order for `skip_if_only_changed` to work?

Reply:
Correct; they are exclusive.  (As the CI helpfully told me in a previous revision :laughing: )

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/25933/pull-ci-openshift-release-master-config/1488975580392067072